src/pages/Browse/index.tsx

```diff
@@ -113,11 +113,14 @@ export class BrowseBase extends React.Component<Props> {
       );
     }
 
-    let globalMessages;
+    let messageMap;
     if (linterMessages && version) {
-      globalMessages =
-        linterMessages[version.selectedPath] &&
-        linterMessages[version.selectedPath].global;
+      messageMap = linterMessages[version.selectedPath];
+    }
+
+    let globalMessages;
+    if (messageMap) {
+      globalMessages = messageMap.global;
```

Comment:
Why not using `messageMap && messageMap.global` below? (since you do that for the `CodeView` prop)